<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptsWidget</class>
 <widget class="QWidget" name="ReceiptsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QPushButton" name="btPrintNew">
       <property name="text">
        <string>Print new</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAutoPrint">
       <property name="toolTip">
        <string>Automatically print receipt when the card is read.</string>
       </property>
       <property name="text">
        <string>Auto print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkThisReaderOnly">
       <property name="toolTip">
        <string>Print only cards read on this application card reader.</string>
       </property>
       <property name="text">
        <string>This reader only</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>When runner is not found, print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lstNotFound"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Receipt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lstReceipt"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Printer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btPrinterOptions">
       <property name="text">
        <string>Printer options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableViewToolBar" name="tblPrintJobsTB" native="true"/>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableView" name="tblCards"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TableView</class>
   <extends>QTableView</extends>
   <header location="global">qf/qmlwidgets/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::TableViewToolBar</class>
   <extends>QWidget</extends>
   <header location="global">qf/qmlwidgets/tableviewtoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblCards</tabstop>
  <tabstop>btPrintNew</tabstop>
  <tabstop>chkAutoPrint</tabstop>
  <tabstop>chkThisReaderOnly</tabstop>
  <tabstop>lstReceipt</tabstop>
  <tabstop>btPrinterOptions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
